hr_skill/views/hr_skill.xml

```diff
@@ -46,22 +41,8 @@
     </record>
 
     <menuitem name="Skills"
-              parent="menu_hr_skill_mgt"
+              parent="hr.menu_human_resources_configuration"
```

Comment:
@feketemihai AFAICT that won't be needed - the menu is only visible if the parent id.